src/allmydata/util/iputil.py

```diff
@@ -88,7 +102,7 @@ def _increase_rlimits():
     increase_rlimits = _increase_rlimits
 
 def get_local_addresses_sync():
-    return _synchronously_find_addresses_via_config()
+    return [native_str(a) for a in _synchronously_find_addresses_via_config()]
```

Comment:
Maybe worth adding a docstring explicitly indicating the return type of this and the other function.